addons/hr/models/hr_employee.py

```diff
@@ -347,16 +347,21 @@ def _get_user_creation_notification_action(message, message_type, next_action):
                     }
                 }
 
+        existing_users_email = self.env['res.users'].search([]).mapped("email")
```

Comment:
```suggestion
        existing_users_email = [record['email'] for record in self.env['res.users'].search_read([], ['email'])]
```
You should use search_read instead of search. So it gives you only id and email record instead whole record set in the search.

Reply:
Thanks, I have made changes.